.github/workflows/default.yml

```diff
@@ -279,3 +279,71 @@ jobs:
         name: pants.deploy.log
         path: .pants.d/pants.log
       if: always()  # We want the log even on failures.
+      
+  windows-conda:
+    needs: [lint, typecheck, test]
```

Comment:
I think `    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/')` is missing here!